### PR TITLE
feat: add WhereIn method

### PR DESCRIPTION
**Pull request by Kamandlou**
## 📑 Description
Add `WhereIn` method

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: contracts/database/orm/orm.go [hunk 1]
```diff
@@ -129,6 +129,8 @@ type Query interface {
 	WithTrashed() Query
 	// With returns a new query instance with the given relationships eager loaded.
 	With(query string, args ...any) Query
+	// WhereIn adds a "where column in" clause to the query.
+	WhereIn(column string, values []any) Query
```
Comment (hwbrzzl) [inline contracts/database/orm/orm.go:133]: S: Please move this method after `Where`.

File: database/gorm/query_test.go [hunk 2]
```diff
@@ -2707,6 +2707,24 @@ func (s *QueryTestSuite) TestWhere() {
...
+			s.Nil(query.Create(&user))
+			s.True(user.ID > 0)
+
+			user1 := User{Name: "where_in_user_1", Avatar: "where_in_avatar_1"}
+			s.Nil(query.Create(&user1))
+			s.True(user1.ID > 0)
+
+			var users []User
+			s.Nil(query.WhereIn("id", []any{user.ID, user1.ID}).Find(&users))
+			s.True(len(users) > 0)
```
Comment (hwbrzzl) [inline database/gorm/query_test.go:2723]:
```suggestion
			s.True(len(users) == 2)
```

**Review by hwbrzzl — COMMENTED** [review]
Looks great, thank Bro! Ci failed, could you please install Mockery and regenerate the interface?

```
go install github.com/vektra/mockery/v2@v2.34.2
mockery --all --keeptree --dir=contracts
```

**Comment by PEMIDI** [comment]
Good job @Kamandlou

**Review by hwbrzzl — APPROVED** [review]
Great job! Thanks for your contribution.

**Pull request merged**